### ISSUE TITLE
libmpv binding

### ISSUE THREAD

**Issue by 6bmwrkjc** [AdeptLanguage/AdeptImport#45]
https://github.com/mpv-player/mpv/tree/master/libmpv

**Comment by IsaacShelton** [comment]
Yes having mpv bindings would be nice. This will be trivial once the first version of Adept 3.x is ready

And luckily, progress is going very fast on it, so this should be possible soon without needing to manually translate everything
